dell-csi-helm-installer/verify.sh

```diff
@@ -449,6 +449,61 @@ function verify_helm_values_version() {
   check_error error
 }
 
+# verify that the authorization sidecar is configured correctly to be used with the authorization proxy server
```

Comment:
this is complex --why not a standalone  script and has curl/wget dependency ?
this script runs from some node A , which installs sidecar on node B and auth Server is on node C
B to C maybe ok and is desired , but A may be firewalled and not alllowed to talk to C ?